extensions/adobe/experience/target/classification.schema.json

```diff
@@ -0,0 +1,93 @@
+    "definitions": {
+        "classification": {
+            "properties": {
+                "xdm:activityName": {
+                    "title": "Activity Name",
+                    "type": "string",
+                    "description": "The name of the Target activity."
+                },
+                "xdm:experienceName": {
+                    "title": "Experience Name",
+                    "type": "string",
+                    "description": "The name of the Target experience."
+                },
+                "xdm:activityExperience": {
+                    "title": "Activity Experience",
+                    "type": "string",
+                    "description": "Target Activity name concatenated with experience name."
+                },
+                "xdm:thirdId": {
+                    "title": "Third Id",
+                    "type": "string",
+                    "description": "The third id of the classification record."
+                },
+                "xdm:activityId": {
+                    "title": "Activity Id",
+                    "type": "string",
+                    "description": "The identity of Target activity."
+                },
+                "xdm:experienceId": {
+                    "title": "Experience Id",
+                    "type": "string",
+                    "description": "The identity of Target experience."
+                },
+                "xdm:analyticsReporting": {
+                    "title": "Analytics Reporting",
+                    "type": "string",
+                    "description": "The reporting type associated with the Target activity."
+                },
+                "xdm:activeDates": {
+                    "title": "Active Dates",
+                    "type": "string",
+                    "description": "The date of Target activity activation."
+                },
+                "xdm:controlExperience": {
+                    "title": "Control Experience",
+                    "type": "string",
+                    "description": "The experience set as control experience in the Target activity."
+                },
+                "xdm:algorithmId": {
+                    "title": "Algorithm Id",
+                    "type": "string",
+                    "description": "The identity of the algorithm used in the Target activity."
+                },
+                "xdm:controlvsTargeted": {
+                    "title": "Control vs Targeted",
+                    "type": "string",
+                    "description": "The traffic allocation type for the Target activity."
+                },
+                "xdm:trafficTypeId": {
+                    "title": "Traffic Type Id",
+                    "type": "string",
+                    "description": "The identity of the traffic type associated with Target activity."
+                }
+            },
+            "required": [
+                "xdm:Key",
```

Comment:
You will need to remove this since you removed the key field.

Reply:
I missed adding this